.github/auto_assign.yml

```diff
@@ -4,13 +4,14 @@ addReviewers: true
 # Set to true to add assignees to pull requests
 addAssignees: false
 
-# A list of reviewers to be added to pull requests (GitHub user name)
+# A list of reviewers to be added to pull requests (GitHub user name) or Github Group Name
```

Comment:
I imagine `team` is meant here? The auto_assign config itself has a reference to `groups` but it's [internally defined](https://github.com/DispatchHealth/auto-assign/blob/ca75861a78e6cf27ff0637fea06090f16035ece6/.github/auto_assign.yml#L22-L27)
```suggestion
# A list of reviewers to be added to pull requests (GitHub user name) or Github Team
```

Reply:
Team is meant